portfolio/src/main/webapp/script.js

```diff
@@ -106,32 +104,105 @@ function createCommentElement(comment) {
 
   return commentElement;
 }
-async function displayComments() {
+document.addEventListener('DOMContentLoaded', () => {
+  document.getElementById('maxNum').addEventListener('input', () => {
+    let parentElement = document.getElementById('comments');
+    while (parentElement.firstChild) {
+      parentElement.removeChild(parentElement.lastChild);
+    }
+    cursorStr = null;
+    displayComments(cursorStr);
+  });
+});
+
+let cursorStr = null;
+async function displayComments(cursorString) {
```

Comment:
reading this, it seems like the assumption is this will only ever be called with latest "next" call?